src/Uno.Extensions.Templates/Uno.Extensions.Templates.csproj

```diff
@@ -19,55 +19,75 @@
+				PackagePath="content/unoapp-extensions/MyExtensionsApp.Skia.Gtk/Images/StoreLogo.png"
+				Pack="true" />
+		<Content Include="content/unoapp-extensions/MyExtensionsApp.Windows/app.manifest"
+				PackagePath="content/unoapp-extensions/MyExtensionsApp.Skia.Linux.FrameBuffer/app.manifest"
+				Pack="true" />
+		<Content Include="content/unoapp-extensions/MyExtensionsApp.Windows/Images/StoreLogo.png"
+				PackagePath="content/unoapp-extensions/MyExtensionsApp.Skia.Linux.FrameBuffer/Images/StoreLogo.png"
+				Pack="true" />
+		<_AllCSProjFiles Include="content/**/*.csproj" />
+		<_AllCSProjFiles Include="content/**/Directory.Packages.props" />
 	</ItemGroup>
 
-	<ItemGroup>
-		<Compile Remove="content/**/*.cs" />
-		<None Include="content/**/*.cs" />
-	</ItemGroup>
-	
-	<Target Name="ReplaceTemplateCode" BeforeTargets="BeforeBuild">
-		<ItemGroup>
-			<Content Include="content\unoapp-extensions\**" KeepDuplicates="False">
-				<PackagePath>content\unoapp-extensions</PackagePath>
-				<Pack>true</Pack>
-			</Content>
-			<Content Remove="content\unoapp-extensions\MyExtensionsApp.Debugging.sln" />
-		</ItemGroup>
-	</Target>
-
 	<Target Name="ReplaceVersions" BeforeTargets="BeforeBuild">
-		<PropertyGroup>
-			<MSBuildDeveloperNamespace>
-				<Namespace Prefix="x" Uri="http://schemas.microsoft.com/developer/msbuild/2003" />
-			</MSBuildDeveloperNamespace>
-		</PropertyGroup>
 
-		<ItemGroup>   
-			<_AllCSProjFiles Include="content/**/*.csproj" />
-			<_AllCSProjFiles Include="content/**/Directory.Packages.props" />
-		</ItemGroup>
+		<ReplaceFileText Filename="%(_AllCSProjFiles.Identity)"
+					MatchExpression="Version=&quot;255.255.255.255&quot;"
+					ReplacementText="Version=&quot;$(PackageVersion)&quot;" />
 
-		<!-- Legacy projects -->
-		<XmlPoke XmlInputPath="%(_AllCSProjFiles.Identity)" Query="//x:PackageVersion[@Version='255.255.255.255']/@Version" Value="$(PackageVersion)" Namespaces="$(MSBuildDeveloperNamespace)" />
-		<XmlPoke XmlInputPath="%(_AllCSProjFiles.Identity)" Query="//x:PackageReference[@Version='255.255.255.255']/@VersionOverride" Value="$(PackageVersion)" Namespaces="$(MSBuildDeveloperNamespace)" />
-		<XmlPoke XmlInputPath="%(_AllCSProjFiles.Identity)" Query="//x:PackageReference[@Version='255.255.255.255']/@Version" Value="$(PackageVersion)" Namespaces="$(MSBuildDeveloperNamespace)" />
-
-		<!-- SDK Style projects -->
-		<XmlPoke XmlInputPath="%(_AllCSProjFiles.Identity)" Query="//PackageVersion[@Version='255.255.255.255']/@Version" Value="$(PackageVersion)" Namespaces="" />
-		<XmlPoke XmlInputPath="%(_AllCSProjFiles.Identity)" Query="//PackageReference[@Version='255.255.255.255']/@VersionOverride" Value="$(PackageVersion)" Namespaces="" />
-		<XmlPoke XmlInputPath="%(_AllCSProjFiles.Identity)" Query="//PackageReference[@Version='255.255.255.255']/@Version" Value="$(PackageVersion)" Namespaces="" />
+	</Target>
 
+	<Target Name="ResetVersions" AfterTargets="Pack">
```

Comment:
If someone does a build but not a pack, will this cause issues with the version being incorrectly updated?

Reply:
I mean to be fair the weren't being reset before... but I can add AfterBuild to the AfterTargets so that it will reset even if Pack isn't invoked but will continue to execute at the right point when we do pack